src/ai/backend/client/cli/manager.py

```diff
@@ -32,6 +32,21 @@ def status():
         sys.exit(1)
 
 
+@manager.command()
+@click.argument('agents', nargs=-1)
+@click.option('-a', '--with-all-agents', is_flag=True,
+              help='Fetch all agent host information as well.')
```

Comment:
Please update the parameter & cli argument list accordingly to the review comment in the manager PR.

Reply:
Changed parameter to conform to the changes in the manager.